PCA_Comp_Analysis_Dredging.R

```diff
@@ -401,30 +495,36 @@ itime1<-hi_rain_start-nc_start+1
 itime2<-hi_rain_end-nc_start+1
 iselect<-itime1:itime2
 
-##############################################################
+
 # predictand time series
-##############################################################
+
 y<-pc[,pcamode]
 iyrs<-length(y)
-##############################################################
+
 # array to store the projection index time series
 # for all climate variables (organized in columns)
-##############################################################
+
 
 iclimvar<-length(ncvarlist)
 xproj<-array(0,dim=c(iyrs,iclimvar))
 
 ##############################################################
-# loop over climate variables
-#note: this loop must be performed 8 times - 2 different seasons and 4 different PCs.
-#the pc number and season must be changed each time in the code above. each time the loop is
-#run, a unique dataframe for the selected season and pc must be saved.
+# Loop over climate variables
+# Note: this loop must be performed 8 times:
+# 2 different seasons and 4 different PCs.
+# The pc number and season must be changed each time 
+# in the code above. each time the loop is
+# Run, a unique dataframe for the selected season 
+# and pc must be saved.
 ##############################################################
 
-#read in shapefile of hawaii state from local folder to add outline to composite plots
-hawaii <- readOGR("C:/Users/ksanf/Documents/Statistical_DS/state_map_info/dems/coast", "coast_geo_shp")
+#read in shapefile of hawaii state from local folder 
+# to add outline to composite plots
 
-#this loop loops through the list of variables and for each variable: calculates the composite anomaly 
+#OET: problem reading the shp file / or problem usign the function?
+#hawaii <- readOGR(datapath, layer="coast_geo_shp")

```

Comment:
I  think the issue was i had moved the shapefile by itself to a separate folder but it needed supporting files that were in the folder with the original download to work

Reply:
You can remove the comment line on your branch, since it works on your system.